### PR TITLE
Bug 1794493: Update crio.yaml templates with correct values

### PR DESCRIPTION
**Pull request by umohnani8**
Update the templates with the corret defaults from cri-o.

Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
Opening this with the updated template fixes. @haircommander will probably pull this into his PR when he adds in the e2e tests.

**Comment by wking** [comment]
/retitle Bug 1792749: Update crio.yaml templates with correct values

File: templates/master/01-master-container-runtime/_base/files/crio.yaml [hunk 1]
```diff
@@ -130,106 +92,88 @@ contents:
...
-    # ]
+    additional_devices = [
+    ]
 
     # Path to OCI hooks directories for automatically executed hooks.
-    # Note: the default is just /usr/share/containers/oci/hooks.d, but /usr is immutable in RHCOS
-    # so we add /etc/containers/oci/hooks.d as well
     hooks_dir = [
-        "/etc/containers/oci/hooks.d",
+        "/etc/containers/oci/hooks.d",
```
Comment (wking) [inline templates/master/01-master-container-runtime/_base/files/crio.yaml]: injecting trailing whitespace, what!? ;)  Obviously a nit, but maybe worth fixing in whatever source this was pulled from.

File: templates/master/01-master-container-runtime/_base/files/crio.yaml [hunk 2]
```diff
@@ -348,3 +259,4 @@ contents:
 
     # The port on which the metrics server will listen.
     metrics_port = 9537
+
```
Comment (wking) [inline templates/master/01-master-container-runtime/_base/files/crio.yaml]: removing a trailing newline?  I'd rather keep it, but I dunno if Go's YAML parser cares or not.

**Comment by wking** [comment]
/retitle Bug 1794493: Update crio.yaml templates with correct values

This PR is actually fixing 1794493.  We hope that will also fix 1792749.  If 1792749 is in fact fixed by this PR, we can close it as a dup of 1794493 after demonstrating the fix works.

**Comment by kikisdeliveryservice** [comment]
```level=error msg="Error: Error waiting for AMI (ami-02831ae178a35dd3d) to be ready: timeout while waiting for state to become 'available' (last state: 'pending', timeout: 40m0s)"```

/retest

**Comment by kikisdeliveryservice** [comment]
an e2e test is forthcoming so holding to ship with fix.

/hold

File: templates/master/01-master-container-runtime/_base/files/crio.yaml [hunk 3]
```diff
@@ -108,17 +70,17 @@ contents:
...
 
     # If true, SELinux will be used for pod separation on the host.
-    # selinux = true
+    selinux = true
 
     # Path to the seccomp.json profile which is used as the default seccomp profile
     # for the runtime. If not specified, then the internal default seccomp profile
     # will be used.
-    # seccomp_profile = "/etc/crio/seccomp.json"
+    seccomp_profile = ""
```
Comment (ashcrow) [inline templates/master/01-master-container-runtime/_base/files/crio.yaml]: Is it true that explicitly setting to `""` is the same as "not specified"?

**Comment by kikisdeliveryservice** [comment]
Can we close this and defer to #1414 which has these changes plus e2e?

**Comment by umohnani8** [comment]
Closing in favour of https://github.com/openshift/machine-config-operator/pull/1414

**Pull request closed without merging**